config/authentication.py

```diff
@@ -208,6 +209,56 @@ def get_user(self, validated_token):
         return user
 
 
+class HCXAuthentication(JWTAuthentication):
+    def authenticate_header(self, request):
+        return "Bearer"
+
+    def authenticate(self, request):
+        jwt_token = request.META.get("HTTP_AUTHORIZATION")
+        if jwt_token is None:
+            return None
+        jwt_token = self.get_jwt_token(jwt_token)
+
+        try:
+            payload = self.decode_jwt(settings.HCX_CERT_URL, jwt_token)
+        except jwt.exceptions.InvalidSignatureError:
+            raise AuthenticationFailed("Invalid signature")
+        except Exception as e:
+            raise InvalidToken({"detail": str(e)})
+
+        user = self.get_user()
+
+        return user, payload
+
+    def get_jwt_token(self, token):
+        return token.replace("Bearer", "").replace(" ", "")
```

Comment:
```suggestion
        return token.replace("Bearer ", "", 1)
```
and raise TokenError if the result is empty string